adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleInterstitial.m

```diff
@@ -75,12 +74,6 @@ @implementation GADMediationVungleInterstitial {
 }
 
 - (void)requestInterstitialAd {
-  if (!self.desiredPlacement.length) {
```

Comment:
Is there any reason we want to remove it? Looks like it still can be  @""

Reply:
It's ok if it is an empty string. The SDK has been updated to be able to handle these cases and will properly return a failed to load callback when the load is requested.

Reply:
Ah okay! Thank you for the explanation.